client/src/components/Modals/PitchFeedback/styles.scss

```diff
@@ -0,0 +1,64 @@
+@import '../../../styles/colors.scss';
+
+.feedback-modal {
+  .ui {
```

Comment:
Be a little careful when using these selectors, could end up selecting something you don't want